src/main/scala/mesosphere/marathon/state/GroupManager.scala

```diff
@@ -157,7 +162,10 @@ class GroupManager @Singleton @Inject() (
       from <- rootGroup()
       (toUnversioned, resolve) <- resolveStoreUrls(assignDynamicServicePorts(from, change(from)))
       to = GroupVersioningUtil.updateVersionInfoForChangedApps(version, from, toUnversioned)
-      _ = BeanValidation.requireValid(ModelValidation.checkGroup(to, "", PathId.empty, config.maxApps.get))
+      _ = {
+        // TODO AW: test
+        validateOrThrow(V2Group(to))(validator = V2Group.v2GroupWithConfigValidator(config))
```

Comment:
Right. @alexanderweber Can you do that in a separate PR as a preparation? Thanks!

https://github.com/mesosphere/marathon/issues/2426